src/client/dfuse/pil4dfs/int_dfs.c

```diff
@@ -94,12 +94,22 @@
 
 /* the default min fd that will be used by DAOS */
 #define DAOS_MIN_FD         10
+/* a dummy fd that will be used to reserve low fd with dup2(). This is introduced to make sure low
+ * fds are reserved as much as possible although applications (e.g., bash) may directly access
+ * low fd. Once a low fd is freed and available, calls dup2(DAOS_DUMMY_FD, fd) to reserve it.
+ */
+#define DAOS_DUMMY_FD       1001
```

Comment:
This comment is now wrong, and you don't really need a `#define` here.  The value is only used in two places currently and one of them should be `fd_dummy` rather than `DAOS_DUMMY_FD`

Reply:
I assumed a low fd has a higher chance compared with a high fd to be closed by force in a bash script, so I keep "#define DAOS_DUMMY_FD       1001" to use a high fd if possible.
If you do think there is no difference, I can remove "DAOS_DUMMY_FD". Thank you!

Reply:
The value is fine but it's not a DUMMY_FD, fd_dummy is allocated by the kernel using DAOS_DUMMY_FD as a guide.  My point was that when you come to use it then you need to use `fd_dummy` rather than `DAOS_DUMMY_FD`.

Reply:
You are right. I will fix the comments here. Thank you!

Reply:
Thank you! It is fixed now.